connectors/webhook/src/main/java/io/camunda/connector/inbound/model/WebhookConnectorProperties.java

```diff
@@ -111,6 +110,15 @@ public record WebhookConnectorProperties(
         @FEEL
         HMACScope[] hmacScopes,
     WebhookAuthorization auth,
+    @TemplateProperty(
+            id = "responseExpression",
+            label = "Response expression",
+            type = PropertyType.Text,
+            group = "webhookResponse",
+            description = "Expression used to generate the HTTP response",
+            feel = FeelMode.required,
+            optional = true)
+        Function<WebhookResultContext, WebhookHttpResponse> responseExpression,
```

Comment:
It's probably done/planned already, but we might consider updating https://docs.camunda.io/docs/components/connectors/protocol/http-webhook/#response-body-expression

Reply:
Planned, yes.